internal/wasm/interpreter/interpreter.go

```diff
@@ -1738,6 +1754,33 @@ func (ce *callEngine) callNativeFunc(ctx context.Context, callCtx *wasm.CallCont
 				}
 			}
 			frame.pc++
+		case wazeroir.OperationKindTableInit:
```

Comment:
mention where these opcodes are from or double space around "// Handle instructions added in FeatureBulkMemoryOperations and WebAssembly 2.0"